### PR TITLE
Expose the "feature" in the sentences we return

### PR DESCRIPTION
**Pull request by richdouglasevans**

### CODE REVIEW

File: krump/sentence/sentence.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+from krump.sentence.mongo import get_sentences as sentences_for
+from krump.support.collections import pluck
+
+
+# noinspection PyUnusedLocal
+def apostrophe_sentences(request, sentences):
+    # TODO
+    pass
```
Comment (richdouglasevans) [inline krump/sentence/sentence.py]: @lpmi-13 This is the bit to implement.

This function is given tow arguments:

* the `request` which is a simple dictionary
* the `sentences` (from Mongo) which is a list filled with sentence documents.

You need to return a list of sentences of the right shape.

File: krump/sentence/sentence_memory.py [hunk 2]
```diff
@@ -1,21 +0,0 @@
...
-    ]
-)
-
-
-def get_sentences(request):
-    sentences = database[request['feature']]
-    if request['maximum_words'] is not None:
-        sentences = [sentence for sentence in sentences if
-                     len(sentence.split(' ')) <= request['maximum_words']]
-    return sentences[:request['count']]
```
Comment (richdouglasevans) [inline krump/sentence/sentence_memory.py]: I removed this file because it's no longer useful.

File: test/krump/sentence/sentence_tests.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        input_sentences = [
+            "Adam's house is made of gingerbread."
+        ]
+
+        actual_sentences = apostrophe_sentences(request, input_sentences)
+
+        expected_sentences = [dict(
+            sentence="Adam's house is Laura's too.",
+            features=["Adam's", "Laura's"]
+        )]
```
Comment (richdouglasevans) [inline test/krump/sentence/sentence_tests.py]: @lpmi-13 Right, these are the tests that needs to pass. Once they do pass the work is done.

As you see the `apostrophe_sentences` function on line `#54` gets passed a request and a list of sentences. The list of sentences in this case has just one sentence. Two of the words in that sentence have apostrophes.

The list that is returned from the `apostrophe_sentences` function needs to have identified the words with apostrophes and have added them to the output. See lines `#56-59` for what is expected.

File: krump/sentence/sentence_memory.py (hunk 2 above)
Comment (lpmi-13) [inline krump/sentence/sentence_memory.py]: if request(dict(features=apostrophe)):
    features = [it's]

Comment (lpmi-13) [inline krump/sentence/sentence_memory.py]: this was supposed to be a joke about returning the features for your comment above, but it was disgustingly decontextualized.

**Pull request merged**